.azure-pipelines/OneBranchPipeline.yml

```diff
@@ -8,24 +8,16 @@
 # Support:        https://aka.ms/onebranchsup                                   #
 #################################################################################
 
-trigger: none 
-
-parameters: # parameters are shown up in ADO UI in a build queue time
-- name: 'debug'
-  displayName: 'Enable debug output'
-  type: boolean
-  default: false
+trigger: none
```

Comment:
Please consider having two files:  
.azure-pipelines/OneBranchPipeline-Official.yml and .azure-pipelines/OneBranchPipeline-PullRequest.yml. 

At least you need one official pipeline